### DISCUSSION TITLE
make /cmi helpop support bungeecord

### DISCUSSION THREAD

**Discussion by ArtifactyNight** [Zrips/CMI#2954] [Feature Requests]
**Description of feature/improvement request:**
make `/cmi helpop` support bungeecord

---
### Detailed explanation of what it should do, when and in what situation: ###
if staff online in skyblock server but player in survival server, then player need help
so he just execute command `/cmi helpop i need help`, well.. staff can see a message who need help

### Basic explanation as to why this feature is needed and/or useful to have: ###
it's very comfortable for staff

**Comment by WolverStones** [comment]
duplicate #1476
